aws/ami/files/scylla_install_ami

```diff
@@ -121,3 +121,8 @@ if __name__ == '__main__':
             f.write(u'exclude=kernel kernel-devel')
     elif platform.machine() == 'aarch64':
         run('yum -y --enablerepo=centos-kernel update kernel')
+
+    run('yum install -y yum-utils')
+    packages = subprocess.check_output('repoquery --requires --resolve --recursive scylla', stderr=subprocess.STDOUT, shell=True)
+    with open('{}/packages.txt'.format(homedir), 'w') as f:
```

Comment:
Perhaps `scylla-packages.txt` for the name?  @avikivity any preference on the location of this file?

Reply:
Renamed.